### PR TITLE
Create Kubernetes resources for 2.6.x pattern 1 

### LINKED ISSUE #140: Create a Helm Chart for WSO2 API Manager version 2.6.0 deployment pattern 1
**Description:**
Create a Helm Chart for WSO2 API Manager version 2.6.0 deployment pattern 1 [1].

[1]: [Deployment pattern 1](https://docs.wso2.com/display/AM260/Deployment+Patterns#DeploymentPatterns-Pattern1)

**Suggested Labels:**
Type/Task

**Suggested Assignees:**
ThilinaManamgoda

**Affected Product Version:**
WSO2 API Manager 2.6.x

### PR DESCRIPTION
**Pull request by ThilinaManamgoda**
fixes #140

### CODE REVIEW

File: pattern-1/apim-analytics/wso2apim-analytics-service.yaml [hunk 1]
```diff
@@ -23,94 +23,26 @@ spec:
...
+      name: 'rest-api-port-3'
       protocol: TCP
-      port: 6067
+      port: 7071
     -
-      name: analytics10
+      name: 'rest-api-port-4'
       protocol: TCP
-      port: 11003
+      port: 7444
```
Comment (chirangaalwis) [inline pattern-1/apim-analytics/wso2apim-analytics-service.yaml]: @ThilinaManamgoda add a new line at EOF.

File: pattern-1/apim-analytics/wso2apim-analytics-deployment.yaml [hunk 2]
```diff
@@ -68,72 +74,21 @@ spec:
...
+      - name: apim-analytics-conf-worker
         configMap:
-          name: apim-analytics-conf-datasources
-      - name: analytics-persistent-disk
-        persistentVolumeClaim:
-          claimName: wso2apim-with-analytics-apim-analytics-volume-claim
-      - name: data-persistent-disk
-        persistentVolumeClaim:
-          claimName: wso2apim-with-analytics-apim-analytics-data-volume-claim
+          name: apim-analytics-conf-worker
```
Comment (chirangaalwis) [inline pattern-1/apim-analytics/wso2apim-analytics-deployment.yaml]: @ThilinaManamgoda add a new line at EOF.

File: pattern-1/apim/wso2apim-deployment.yaml [hunk 3]
```diff
@@ -79,11 +79,11 @@ spec:
           protocol: "TCP"
```
Comment (chirangaalwis) [inline pattern-1/apim/wso2apim-deployment.yaml]: IMO, we should define the resource requirements and limits for API Manager based on the recommendations [1].

WDYT?

[1]: [Resource requirements and limits](https://github.com/wso2/kubernetes-apim/blob/2.2.x/pattern-1/apim/wso2apim-deployment.yaml#L56)

File: pattern-1/scripts/deploy.sh [hunk 4]
```diff
@@ -114,4 +112,5 @@ sleep 10s
 
 echoBold 'Deploying Ingresses...'
 ${KUBECTL} create -f ../ingresses/wso2apim-ingress.yaml
-${KUBECTL} create -f ../ingresses/wso2apim-analytics-ingress.yaml
+
+echoBold 'Finished'
```
Comment (chirangaalwis) [inline pattern-1/scripts/deploy.sh]: @ThilinaManamgoda add a new line at EOF.

File: pattern-1/extras/confs/rdbms/mysql/dbscripts/init.sql [hunk 5]
```diff
@@ -1946,4 +2007,4 @@ create index IDX_AS_AITIAI on AM_SUBSCRIPTION (API_ID,TIER_ID,APPLICATION_ID);
 create index IDX_APA_QT on AM_POLICY_APPLICATION (QUOTA_TYPE);
 create index IDX_AA_AT_CB on AM_APPLICATION (APPLICATION_TIER,CREATED_BY);
 
--- Performance indexes end--
+-- Performance indexes end--
```
Comment (chirangaalwis) [inline pattern-1/extras/confs/rdbms/mysql/dbscripts/init.sql]: @ThilinaManamgoda add a new line at EOF.

File: pattern-1/apim-analytics/wso2apim-analytics-deployment.yaml [hunk 6]
```diff
@@ -31,7 +31,7 @@ spec:
     spec:
       containers:
       - name: wso2apim-with-analytics-apim-analytics
-        image: docker.wso2.com/wso2am-analytics:2.5.0
+        image: docker.wso2.com/wso2am-analytics-worker:2.6.0
         resources:
```
Comment (chirangaalwis) [inline pattern-1/apim-analytics/wso2apim-analytics-deployment.yaml]: IMO, we should define the resource requirements and limits for API Manager Analytics based on the recommendations [1].

WDYT?

[1]: [Resource requirements and limits](https://github.com/wso2/kubernetes-apim/blob/2.2.x/pattern-1/apim-analytics/wso2apim-analytics-deployment.yaml#L56)

File: pattern-1/apim/wso2apim-deployment.yaml (hunk 3 above)
Comment (ThilinaManamgoda) [reply pattern-1/apim/wso2apim-deployment.yaml]: Yes, this is a good idea. Let's try to add this if time permits.

**Pull request merged**